projects/packages/jetpack-mu-wpcom/src/features/launch-button/index.js

```diff
@@ -0,0 +1,11 @@
+import { wpcomTrackEvent } from '../../common/tracks';
+
+document.addEventListener( 'DOMContentLoaded', () => {
+	const launchButton = document.querySelector( '#wpadminbar .launch-site' );
+	if ( ! launchButton ) {
+		return;
+	}
+	launchButton.addEventListener( 'click', () => {
+		wpcomTrackEvent( 'wpcom_adminbar_launch_site' );
```

Comment:
I'm not entirely sure this is working. Is there a way to test this properly and to see the events arriving to our dashboards without merging this PR?

Reply:
This is a good question. :)

Reply:
Also, what about just adding an `onclick` html attribute?

Reply:
> Also, what about just adding an onclick html attribute? And avoid all the enqueuing for just this.

I would love to be able to do that :) but I don't know how specifically because of the `/common/tracks` dependency.

Reply:
One way to test this is by observing the `t.gif` requests in the Networks tab of the browser devtools (remember to enable the "Preserve log" option since there is a redirection after clicking in the button):

<img width="1266" alt="Screenshot 2025-01-30 at 13 35 34" src="https://github.com/user-attachments/assets/6fa9732c-fc97-4114-b75f-615fdd53ba20" />

Eventually (after ~5/10 minutes) the recorded event should appear in the Tracks Live View (see PCYsg-5hv-p2).